bitnami/apache/2.4/debian-11/rootfs/opt/bitnami/scripts/apache/setup.sh

```diff
@@ -61,3 +61,6 @@ fi
 
 # Fix logging issue when running as root
 ! am_i_root || chmod o+w "$(readlink /dev/stdout)" "$(readlink /dev/stderr)"
+
+# Allow to know when setup is done (used by startupProbe)
+date > /tmp/bitnami-setup-done
```

Comment:
Another option (instead of create the file `/tmp/bitnami-setup-done`) is to start mysql in a different port (so the health-check will fail) and after the setup and the stop, start it again with the right port

Reply:
I released how you fix this problem in other containers. For example in mysql, during the setup, you start the process with `--bind-address=127.0.0.1`, so the external connection should fails.

This can be improved using a different port. In apache there isn't a bg process during the setup so this change is no necessary.